2021q3/sdhci_acpi.adoc

```diff
@@ -0,0 +1,19 @@
+=== Marvell SDHCI improvements and ACPI support
+
+Contact: Bartlomiej Grzesik <bag@semihalf.com> +
+Contact: Marcin Wojtas <mw@semihalf.com>
+
+The Semihalf team was working on the ACPI support for the
```

Comment:
If you're still working on that, s/was/has been/. Ignore otherwise.

Reply:
This is done.